src/main/java/io/confluent/connect/jdbc/dialect/Db2DatabaseDialect.java

```diff
@@ -62,7 +62,7 @@ public Db2DatabaseDialect(AbstractConfig config) {
 
   @Override
   protected String currentTimestampDatabaseQuery() {
-    return "values(CURRENT_TIMESTAMP)";
+    return "SELECT CURRENT_TIMESTAMP FROM SYSIBM.SYSDUMMY1;";
```

Comment:
IIUC, `CURRENT_TIMESTAMP` is equivalent to `CURRENT TIMESTAMP(6)`, which might be for example `2015-03-23-09.41.24.684842` ([ref](https://www.ibm.com/support/knowledgecenter/en/SS6NHC/com.ibm.swg.im.dashdb.sql.ref.doc/doc/r0005886.html)). Is that correct? Is this the precision we want to use?